studies/awkward_cupy_perfscript.py

```diff
@@ -1,12 +1,12 @@
 import awkward as ak
```

Comment:
This file (being in the studies/ directory) didn't need to be changed, but it's not bad to drop all the "._v2"s.